tasks/main.yml

```diff
@@ -1,11 +1,5 @@
 ---
 # Homebrew setup prerequisites.
-- name: Get current account's username.
```

Comment:
Can you pull the whoami-->ansible_user_id change into a separate PR? I'd be willing to merge that ASAP (before I get a chance to fully review this PR).